apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/db/DatabaseManagerTest.java

```diff
@@ -142,7 +142,7 @@ void createTemporaryDb() throws Exception {
 
     @Test
     void testFindFullDatasources() {
-        Collection<TransactionalDataSource> fullDatasources = ((ShardManagement) databaseManager).getFullDataSources();
+        Collection<TransactionalDataSource> fullDatasources = ((ShardManagement) databaseManager).getFullDataSources(2L);
```

Comment:
It ids good idea to replace 2L with explicitly declared constant

Reply:
It's unit test. No sense here...